rust/tests/all_test.rs

```diff
@@ -136,9 +136,9 @@ fn benchmark_test() {
     let loops = 100000;
     let mut bd: Vec<BenchmarkData> = Vec::new();
     for _i in 0..loops {
-        let lat = rng.gen_range(-90.0, 90.0);
-        let lng = rng.gen_range(-180.0, 180.0);
-        let mut len = rng.gen_range(2, 15);
+        let lat = rng.gen_range(-90.0..90.0);
```

Comment:
It looks like the prevailing convention with gen_range is to pass in the interval endpoints rather than a range. Unless this is backwards-compatible and there's some recommendation to change style in new usages, mind leaving as it was?

Reply:
It is actually more forceful than just a style recommendation. The underlying rand crate [changed the function declaration](https://github.com/rust-random/rand/pull/1003) so only the new version will even compile.